[influxdb] invalid states on startup

When using a `restoreOnStartup` strategy with influxdb, the log is full of errors like these:

```
2017-07-06 10:04:16.867 [ERROR] [rthome.core.library.items.NumberItem] - Tried to set invalid state 6799.360000000001 on item weather_brightness_max of type NumberItem, ignoring it
```

This happens for all of my group items. As it turns out, it tries to set StringItems because GroupItem does not match any of [these](https://github.com/openhab/openhab1-addons/blob/master/bundles/persistence/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/internal/InfluxDBPersistenceService.java#L504). 

This happens on OH 2.1 and OH 2.2-SNAPSHOT build #980.

--- a/bundles/persistence/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/internal/InfluxDBPersistenceService.java
+++ b/bundles/persistence/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/internal/InfluxDBPersistenceService.java
@@ -26,6 +26,7 @@ import org.influxdb.dto.Pong;
 import org.influxdb.dto.Query;
 import org.influxdb.dto.QueryResult.Result;
 import org.influxdb.dto.QueryResult.Series;
+import org.openhab.core.items.GroupItem;
 import org.openhab.core.items.Item;
 import org.openhab.core.items.ItemNotFoundException;
 import org.openhab.core.items.ItemRegistry;
@@ -501,6 +502,9 @@ public class InfluxDBPersistenceService implements QueryablePersistenceService {
         if (itemRegistry != null) {
             try {
                 Item item = itemRegistry.getItem(itemName);
+                if (item instanceof GroupItem) {
+                    item = ((GroupItem) item).getBaseItem();
+                }
                 if (item instanceof ColorItem) {
                     logger.debug("objectToState found a ColorItem {}", valueStr);
                     return new HSBType(valueStr);

